ign-gui6.yaml

```diff
@@ -0,0 +1,41 @@
+repositories:
+  ign-cmake:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-cmake
+    version: ign-cmake2
+  ign-common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: ign-common4
+  ign-gui:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-gui
+    version: main
+  ign-math:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-math
+    version: ign-math6
+  ign-msgs:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-msgs
+    version: ign-msgs7
+  ign-plugin:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-plugin
+    version: ign-plugin1
+  ign-rendering:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-rendering
+    version: ign-rendering5
+  ign-tools:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-tools
+    version: ign-tools1
+  ign-transport:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-transport
+    version: ign-transport10
+  ign-utils:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-utils
+    version: ign-utils1
```

Comment:
I don't see this listed as a dependency in https://github.com/ignitionrobotics/ign-gui/blob/main/.github/ci/packages.apt (also didn't see it in the `gui5` branch), should we add it in there?

Reply:
The tricky thing about `gazebodistro` is that we need to list indirect dependencies as well. See https://github.com/ignition-tooling/gazebodistro/pull/28, this is here because `ign-gui` depends on `ign-utils` indirectly through `ign-transport`.

Reply:
I didn't know about the indirect dependencies rule here. Thanks, that makes sense, so we can mark this as resolved :+1: